### PR TITLE
Define CDP interfaces that mapper needs

### PR DESCRIPTION
**Pull request by OrKoN**
This PR makes it explicit which CDP interfaces
the mapper implementation expects.

### CODE REVIEW

File: src/bidiMapper/commandProcessor.ts
Comment (sadym-chromium) [inline src/bidiMapper/commandProcessor.ts:26]: go/tsstyle#identifiers-abbreviations

File: src/bidiMapper/domains/context/browsingContextProcessor.ts
Comment (thiagowfx) [inline src/bidiMapper/domains/context/browsingContextProcessor.ts:331]:
```suggestion
    const client = params.cdpSession;
```

File: src/bidiMapper/domains/context/browsingContextProcessor.ts
Comment (OrKoN) [inline src/bidiMapper/domains/context/browsingContextProcessor.ts:331]:
```suggestion
    const client = params.cdpSession
```

File: src/bidiMapper/domains/context/browsingContextProcessor.ts
Comment (OrKoN) [reply src/bidiMapper/domains/context/browsingContextProcessor.ts:331]: suggestion does not work :) it's a ternary operator.

Comment (thiagowfx) [reply src/bidiMapper/domains/context/browsingContextProcessor.ts:331]: I'm sorry, the diff confused me ><

**Review by thiagowfx — APPROVED** [review]
Please wait for @sadym-chromium's approval on this one

**Review by sadym-chromium — APPROVED** [review]

**Pull request merged**
